dts/bindings/sensor/maxim,max31875.yaml

```diff
@@ -0,0 +1,32 @@
+description:
+  Maxim MAX31875 Low-Power I2C Temperature Sensor.
+  Find the datasheet here https://datasheets.maximintegrated.com/en/ds/MAX31875.pdf
+
+compatible: "maxim,max31875"
+
+include: "i2c-device.yaml"
+
+properties:
+  conversions-per-second:
+    description: Number of temperature readings performed by the MAX31875 per second.
```

Comment:
All use of defaults must be justified in the description. Please read https://docs.zephyrproject.org/latest/guides/dts/bindings.html#rules-for-mainline-bindings in detail for all the rules about writing bindings for submission upstream.